packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -224,6 +229,11 @@ export interface IColumn {
   iconName?: IconName;
 
   /**
+   * Whether or not only the icon is used in the column header.
+   */
+  isIconOnly?: boolean;
```

Comment:
it would be nice to understand from the comment why this property matters. What behavior does it control?